src/client/sandbox/index.js

```diff
@@ -18,6 +18,8 @@ import XhrSandbox from './xhr';
 import StorageSandbox from './storages';
 import { isIE, isWebKit } from '../utils/browser';
 import { create as createSandboxBackup, get as getSandboxBackup } from './backup';
+import { get } from '../utils/destination-location';
```

Comment:
You shouldn't export single method, since it's lacks context completely. Export `destinationLocation` as a whole